### DISCUSSION TITLE
Некоторые замечания по UI

### DISCUSSION THREAD

**Discussion by enomado** [Karasiq/webm-tv#7] [Ideas]
Сходу:

1. хитбокс крутилки слишком маленький, в него тяжело попадать мышкой
2. от видео к видео меняется размер страницы, и кнопка следущее видео уезжает

А так проект отличный

**Comment by enomado** [comment]
А стрелками нужно проматывать на 10 сек, а не запускать следущее видео

**Comment by SirMaksimRuban** [comment]
Мне до@баться не до чего, просто спасибо, юзаю уже много лет.

**Comment by Karasiq** [comment]
> Сходу:
> 
>     1. хитбокс крутилки слишком маленький, в него тяжело попадать мышкой
> 
>     2. от видео к видео меняется размер страницы, и кнопка следущее видео уезжает
> 
> 
> А так проект отличный

Так можно использовать стрелки/жесты мышью и Shift+L для зацикливания.
